src/client.c

```diff
@@ -44,10 +45,14 @@ urlinfo_t *parse_url(char *url)
     5. Set the port pointer to 1 character after the spot returned by strchr.
     6. Overwrite the colon with a '\0' so that we are just left with the hostname.
   */
+  path = (strchr(hostname, '/') + 1);
```

Comment:
Nice, very clean, concise code here. Some might tell you to draw it out more, but I like the compact format. The judgment will depend on who you work for.